geomstats/geometry/beta_distributions.py

```diff
@@ -221,12 +237,17 @@ def exp(self, tangent_vec, base_point, n_steps=N_STEPS):
         Parameters
         ----------
         tangent_vec : array-like, shape=[..., dim]
+            Tangent vector.
         base_point : array-like, shape=[..., dim]
+            Base point.
         n_steps : int
+            Number of steps for integration.
+            Optional, default: N_STEPS.
```

Comment:
maybe write the value of N_STEPS otherwise it won't appear in the doc but only in the code?

Reply:
Yes, I hesitated on whether to put the actual value too. The risk with the actual value is that we might forget to change the docstring when we update the value of N_STEPS.

I have changed it though, let's try to think about it.